packages/hardhat-plugin/package.json

```diff
@@ -36,6 +36,13 @@
     "directory": "packages/hardhat-plugin"
   },
   "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "2.0.0",
```

Comment:
Should we list them as peer dependencies?

Reply:
Yes it will be better to have it optional. This requires a bit of changes to use ethers provider directly as well as contracts function (vs the one provided by the hardhat-ethers plugin currently in use in our lib)